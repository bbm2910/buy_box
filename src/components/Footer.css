footer {
	background-color: #e5e5e5;
	font-size: 0.8rem;
	padding: 5px;
	text-align: center;
	position: fixed;
	bottom: 0;
	width: 100%;
}
