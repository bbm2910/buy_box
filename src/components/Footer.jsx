import './Footer.css';
const Footer = () => {
	return <footer>Â© BM 2024</footer>;
};

export default Footer;
