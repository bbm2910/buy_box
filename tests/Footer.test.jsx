import { render, screen } from '@testing-library/react';
import Footer from '../src/components/Footer';

describe('Footer component', () => {
	it('renders the footer with copyright text', () => {
		const { container } = render(<Footer />);
		expect(container).toMatchSnapshot();
		const footerElement = screen.getByText(/Â© BM 2024/i);
		expect(footerElement).toBeInTheDocument();
	});
});
