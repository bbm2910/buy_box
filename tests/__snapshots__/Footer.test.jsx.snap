// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > renders the footer with copyright text 1`] = `
<div>
  <footer>
    Â© BM 2024
  </footer>
</div>
`;
